schemas/rum/view-schema.json

```diff
@@ -73,6 +73,12 @@
               "minimum": 0,
               "readOnly": true
             },
+            "interaction_to_next_paint": {
+              "type": "integer",
+              "description": "Duration in ns of the longest interaction to the next paint",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
              "description": "Longest duration in ns between an interaction and the next paint",
```